### PR TITLE
Test Point2Point2Range as in RoME

### PR DESCRIPTION
**Pull request by Affie**
Basically the same test that fails in https://github.com/JuliaRobotics/RoME.jl/issues/618

### CODE REVIEW

File: src/services/EvalFactor.jl [hunk 1]
```diff
@@ -182,13 +182,13 @@ function computeAcrossHypothesis!(ccwl::Union{<:CommonConvWrapper{F},
...
       addEntr = view(ccwl.params[sfidx], allelements[count])
-      # dynamic estimate with user requested speadNH of how much noise to inject (inflation or nullhypo)
-      spreadDist = calcVariableDistanceExpectedFractional(ccwl, sfidx, certainidx, kappa=ccwl.inflation)
       
       # do proposal inflation step, see #1051
       # consider duplicate convolution approximations for inflation off-zero
       # ultimately set by dfg.params.inflateCycles
       for iflc in 1:inflateCycles
+        # dynamic estimate with user requested speadNH of how much noise to inject (inflation or nullhypo)
+        spreadDist = calcVariableDistanceExpectedFractional(ccwl, sfidx, certainidx, kappa=ccwl.inflation)
```
Comment (dehann) [inline src/services/EvalFactor.jl:191]: I had it consistent over inflate cycles to ensure inflation stays strong rather than anneal down, are you trying to reduce the "noise" on range only results, or increase the inflation strength?  Perhaps just give a bit more detail on why this change?

**Review by dehann — CHANGES_REQUESTED** [review]

File: src/services/EvalFactor.jl (hunk 1 above)
Comment (Affie) [reply src/services/EvalFactor.jl:191]: increase the inflation strength: I do not know what changed, but it doesn't get there with a reasonable amount of `inflateCycles` 
But at least we know the problem has something to do with inflation.

Comment (Affie) [reply src/services/EvalFactor.jl:191]: If the variance is small to start with `spreadDist` is small and inflation takes a long time to "walk around the circle"

Comment (dehann) [reply src/services/EvalFactor.jl:191]: thanks Johan you're right

**Review by dehann — APPROVED** [review]
Go for merge, thanks
